### PR TITLE
More variety in the hails

### PR DESCRIPTION
**Pull request by Pointedstick**
~~Re-did the band name and song title sections, as requested by @Wrzlprnft~~ (update: too controversial; will re-do and try again later). Also added more content to other sections, and removed a few awkward phrasings.

The best way to view the diff is to look at the second commit, not the first--which is simply a formatting change to decrease the error rate when writing new entries.

### CODE REVIEW

**Comment by Wrzlprnft** [comment]
On phone it is hard to see what you did.
I am not sure, are the band names and the songs now mixed up? Because that would be bad, since the story about "There might be riots" clearly gives them sad archie and gluttony.

**Comment by Wrzlprnft** [comment]
But: thanks. Even with online band name generator and songname generator i did not get very satisfying results like these.

**Comment by jafdy** [comment]
The band names and songs are mixed up, so it is possible for someone other than TMBR to write sad archie or gluttony.

**Comment by KiLEdEnNis** [comment]
@jafdy maybe copyright issues? lul

File: data/hails.txt [hunk 1]
```diff
@@ -8,244 +8,376 @@
...
-		`"Under-paid Bandit" `
-	phrase
-		"musical venue"
+		". "
 	word
-		"The song "
-		"Their song "
+		"Their music"
+	word
+		" "
```
Comment (KiLEdEnNis) [inline data/hails.txt]: small tip (besides every band owning every song)
putting a space bar after "Their music" (to mke it "Their music " can save a word being only " "
don't know if you know this & it works either way, but can save some lines

**Comment by KiLEdEnNis** [comment]
not sure what happened there (that Pull Request Review), blame my laptop

**Comment by Pointedstick** [comment]
Yes, they are mixed up now. I didn't think it added much to have specific songs associated with bands, and instead I just added way more bands and song names to both lists. But I can take off Sad Archie and Gluttony if you think anybody will notice.

**Comment by KiLEdEnNis** [comment]
I would take all the new names and put them in a file, then add song names / bands to the base file (so just adding the new names / bands)

**Comment by endless-sky** [comment]
I like the huge new list of song names, but I do prefer having specific songs linked to each band. Maybe we could have a vote for, say, the ten coolest-sounding band names, then split up the song names among them?

**Comment by Pointedstick** [comment]
Sounds good to me. We could probably re-use some of the band names as song names anyway.

**Comment by Pointedstick** [comment]
Actually, that presents a problem. Coupling the band names to their song titles in the provided data structures, as was done before, reduces the flexibility to write hails that involve only band names, or only song titles, or band names and song titles in different configurations than what would be provided by a `phrase "band name and song title"` data structure.

**Comment by jafdy** [comment]
You could keep both. Have a "song" and "band" lists for any hail that only mentions a song or a band but not both, and have the old "band and song" list for the hails where the both a band and their song are mentioned.

**Comment by Wrzlprnft** [comment]
That may lead to certain band names appearing way more often than others

**Comment by KiLEdEnNis** [comment]
@Wrzlprnft some bands are more popular than others maybe? :P

**Comment by Wrzlprnft** [comment]
Yeah, the problem is still: let's say bands are one of 8 options of smalltalk.
Then, the game can choose between random kombinations and not random. And we have one important band.
So at the moment half of all band talk would be TMBR, which would be 1/8*1/2 of all.

**Comment by KiLEdEnNis** [comment]
then you would neet to copy the random part several times :/
hails should get a frequency like fleet variants
so you would make the frequency of the random bands 6x as high, so TMBR is only 1,33x more frequent

**Comment by Pointedstick** [comment]
It occurs to me that a possible way out of this conundrum is simply to remove TMBR's iconic songs from the band list; that way there's never a chance that another band will be erroneously credited with their songs. Then the band names and song titles can all be random and separate from one another.

I think this works much better than having certain band names associated with certain bands, for a verity of reasons: it allows you to have far more bands; the code is much cleaner and less incorrect; and content creators have more flexibility to write different hail messages that use band names, songs, or both, and in whatever order they like.

**Comment by Bladewood** [comment]
It's not good if the TMBR missions become the only times you ever hear Sad Archie and the other songs.

**Comment by Pointedstick** [comment]
If that's the stumbling block, I could add references to them elsewhere.

**Comment by Pointedstick** [comment]
I removed the new band stuff since it was too controversial. I'll re-work it in another branch. Hopefully this is merge-able now.

**Comment by endless-sky** [comment]
Looks good. Thanks for all the new additions!

**Pull request merged**